add new suru type for featured blog posts section

sowasred2012 opened the pull request:
## Done

- Added a new suru type for use on the featured posts section of the blog index

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/blog
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the featured section suru matches the [design](https://github.com/canonical-web-and-design/web-squad/issues/1877#issuecomment-550279581).


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1877

## Screenshots

![FireShot Capture 015 - Ubuntu Blog - Ubuntu - localhost](https://user-images.githubusercontent.com/2376968/68322029-99a70b80-00ba-11ea-8f74-7b57869be6ac.png)

webteam-app commented:
Starting demo at: https://ubuntu-com-canonical-web-and-design-pr-6080.run.demo.haus/

anasereijo commented:
@sowasred2012 this is looking great! 
Is there a way to fix suru just on mobile version? The suru folds look very steep.
It's also making those weird lines.

![Screenshot 2019-11-07 at 10 18 56](https://user-images.githubusercontent.com/36884067/68380697-3f548c00-0148-11ea-9bc1-c78673c369b0.png)

anasereijo commented:
Looks good to me 👍 

We should put this in our suru documentation. Not sure if should be a new strip or another version of the `p-strip--suru`. Any thoughts? @pmahnke

anasereijo commented:
@solazio ready for code review

sowasred2012 commented:
@anasereijo @pmahnke I've added them to the docs, let me know if you want me to change anything. Although it looks fairly similar, the use case felt pretty different to the standard suru, so I've added it as a new one.

solazio commented on static/sass/_pattern_strip.scss:
```diff
@@ -8,6 +8,7 @@
   @include ubuntu-p-strip-suru-half-and-half;
   @include ubuntu-p-strip-suru-background;
   @include ubuntu-p-strip-suru-half-top;
+  @include ubuntu-p-strip-suru-blog-hero;
```
aren't these suppose to be ordered alphabetically?
